File: 03.business-os/_archive/commonos_legacy_skeleton_20260425_184835/app_local/PocketSecretary/iphone/PocketSecretary/CommonOSRuntime/CommonComponentUsage.swift
import Foundation

enum CommonComponentUsage {
    static func createDefault() -> CommonOSRuntimeDescriptor {
        CommonOSRuntimeDescriptor(
            appName: "PocketSecretary",
            commonComponentUsage: [
                "Card",
                "List",
                "Search Bar",
                "Filter Panel",
                "Status Chip",
                "Toast",
                "Offline Queue Status UI",
                "Sync Retry UI",
                "Conflict Review UI",
                "App Shell"
            ],
            featureVariants: [
                "panel.sync",
                "panel.conflict",
                "input.compact",
                "button.primary",
                "card.standard"
            ]
        )
    }
}

File: 03.business-os/PocketSecretary/android/app/src/main/java/com/lsam/PocketSecretary/runtime/commonos/CommonComponentUsage.java
package com.lsam.PocketSecretary.runtime.commonos;

import java.util.ArrayList;
import java.util.List;

public final class CommonComponentUsage {

    public CommonOSRuntimeDescriptor createDefault() {
        final List<String> components = new ArrayList<>();
        components.add("Card");
        components.add("List");
        components.add("Search Bar");
        components.add("Filter Panel");
        components.add("Status Chip");
        components.add("Toast");
        components.add("Offline Queue Status UI");
        components.add("Sync Retry UI");
        components.add("Conflict Review UI");
        components.add("App Shell");

        final List<String> variants = new ArrayList<>();
        variants.add("panel.sync");
        variants.add("panel.conflict");
        variants.add("input.compact");
        variants.add("button.primary");
        variants.add("card.standard");

        return new CommonOSRuntimeDescriptor("PocketSecretary", components, variants);
    }
}
